packages/trader/src/Utils/React/deeper-compare.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Deep compare component's props and state for re-render check.
+ *
+ * @return {Boolean} Whether we should re-render the component or not
+ */
+export const deeperCompare = (next_props, next_state, this_props, this_state) => {
```

Comment:
is this actually faster than just rerendering? because `deeperCompare` seems quite heavy.

Reply:
Yeah, this is only used in `PurchaseFieldset` which is one of our heavy components (30ms each rerender, which is way over 16ms budget).